src/test/scala/sss/db/DbV2Spec.scala

```diff
@@ -240,6 +215,16 @@ class DbV2Spec extends DbSpecSetup {
     } finally fixture.dbUnderTest.dropView("testview2").runSyncAndGet
   }
 
+  it should "support select query with non empty base where statement" in {
```

Comment:
Is this test enough or should I add more? If so,  should I create a separate QuerySpec?

Reply:
This is reasonable. Ive asked to add the baseWhere to View also. So if maybe QuerySpec? Up to you.